ci/Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'etc'
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "base"
+
+  if Vagrant.has_plugin?('vagrant-env')
+    config.env.enable
+  end
+
+  machine_name = ENV['MACHINE_NAME'] || ("vmck-vagrant-" + `hostname`.strip)
+  custom_sh = ENV['PROVISION_SH']
+
+  config.vm.define machine_name
+  config.nfs.functional = false
+
+  config.vm.synced_folder "..", "/opt/vmck", type: "rsync",
+      rsync__exclude: [".git/", "data/", "vmck.egg-info/", ".pytest_cache"]
+
+  config.vm.provision(
+      'provider',
+      preserve_order: true,
+      type: 'shell',
+      path: "wait-cluster.sh",
+      inline: nil,
+      privileged: false,
+    )
```

Comment:
This bit looks like it was copied over from https://github.com/liquidinvestigations/node/blob/master/vagrant/Vagrantfile#L39, where it was so complicated because it was overriding the provisioner named `provider`, which was intended to be specific to the vagrant provider (virtualbox, libvirt, vmck, etc). For us, our only provider is `vmck`, so you can simplify this line a lot. Take a look at the vagrant docs, examples online, etc, and see if you can reduce it to a single short line.